notes/coredns-1.11.0.md

```diff
@@ -3,53 +3,77 @@ title = "CoreDNS-1.11.0 Release"
 description = "CoreDNS-1.11.0 Release Notes."
 tags = ["Release", "1.11.0", "Notes"]
 release = "1.11.0"
-date = "2023-06-02T00:00:00+00:00"
+date = "2023-07-25T00:00:00+00:00"
 author = "coredns"
 +++
 
+## Release Highlights
+
+* Adds support for accepting DNS connections over QUIC (doq).
+* Adds CNAME target rewrites to the _rewrite_ plugin.
+* Plus many bug fixes, and some security improvements.
+
```

Comment:
```suggestion

This release introduces the following backward incompatible changes:
* In the _kubernetes_ plugin, we have dropped support for watching Endpoint and Endpointslice v1beta, since all supported K8s versions now use Endpointslice.
* The _bufsize_ plugin changed its default size limit value to 1232
* Some changes to _forward_ plugin metrics.
```